Add example table for grace period (#436)
bufbuild/docs.buf.build: docs/subscription/index.md

### Does Buf offer overage protection?

To help protect our customers from unexpected overages, Buf offers the option for billing admins to receive an email
alert whenever the usage for their organization has progressed into the next level. These email alerts are enabled by

> default and will be sent when usage has passed the next increment of 100 Types. Although we will **not** block uploading

Reviewer: ```suggestion
default and are sent when usage exceeds the next increment of 100 Types. Although we will **not** block uploading
```

of more Types, admins will have the ability to discuss with their organization and adjust if necessary.

If admins wish to receive alert emails but prefer to decrease noise, a higher threshold can be set before messages are
sent. Assume a customer is on the Pro plan and has a desired budget of $500 USD. The alert threshold can be set at 500
Types, and no alerts will be sent until they reach that level of usage. An email alert will be sent at each level beyond
the chosen threshold.

> Additionally, a grace period of 1 billing cycle is automatically applied before customers are invoiced for a higher

Reviewer: ```suggestion
Additionally, a grace period of 1 billing cycle is automatically applied before you are invoiced for a higher
```

amount. In practice, the grace period means you will be invoiced for the lower number of max types between current and
previous billing cycles. To illustrate the grace period, see the table below.

### Usage Example

> Assume a customer subscribes to the Pro plan in March.

Reviewer: ```suggestion
Assume you subscribe to the Pro plan in March.
```
